package.json

```diff
@@ -32,7 +32,8 @@
     "vscode-languageserver": "6.1.1",
     "vscode-languageserver-protocol": "3.15.3",
     "vscode-languageserver-textdocument": "1.0.8",
-    "vscode-languageserver-types": "3.17.3"
+    "vscode-languageserver-types": "3.17.3",
+    "graphql": "16.7.1"
```

Comment:
Need to look into this more.
The issue is that graphql-armor/cost-limit gets a different version of graphql
https://github.com/Escape-Technologies/graphql-armor/blob/b4208f065b6a6ca2b650077b177dd8360808ed8b/packages/plugins/cost-limit/package.json#L30